packages/manager/apps/hpc-vmware-managed-vcd/public/translations/terminate/Messages_fr_FR.json

```
@@ -0,0 +1,4 @@
+{
+  "terminate_managed_vcd_success": "Votre demande de résiliation a bien été prise en compte. Un e-mail contenant la procédure vous a été envoyé.",
```

```suggestion
  "terminate_managed_vcd_success": "Votre demande de résiliation a bien été prise en compte. Un email contenant la procédure vous a été envoyé.",
```

Ask to Thomas if you want more explanation 😅